src/applications/gi/tests/e2e/00-required.cypress.spec.js

```diff
@@ -11,11 +11,11 @@ import { mockTogglesResponse } from './mock-feature_toggles';
 
 const institutionAttributes = institutionProfile.data.attributes;
 
-describe('Institution', () => {
+describe.skip('Institution', () => {
```

Comment:
@dan-brooking , is it intentional that these tests are being skipped? Will they be restored?

Reply:
@timcosgrove yes, these will be removed once everything makes it to prod fully. Apparently cypress tests run against prod and at the time so it all needs to be on prod before these pass.